powermon/commands/command_definition.py

```diff
@@ -0,0 +1,51 @@
+from powermon.protocols import ResponseType
+from powermon.dto.command_definition_dto import CommandDefinitionDTO
+
+class CommandDefinition:
+    def __init__(self, code, description, help_text: str, response_type : ResponseType, 
+                 responses, test_responses: list[bytes], regex: str, type: str):
+        if responses is None:
+            raise ValueError(f"responses cannot be None for command_code: {code}")
+        if test_responses is None:
+            raise ValueError(f"test_responses cannot be None for command_code: {code}")
+        self.code = code
+        self.description = description
+        self.help_text = help_text
+        self.response_type : ResponseType = response_type
+        self.responses = responses
+        self.test_responses : list[bytes] = test_responses
+        self.regex : str | None = regex
+        self.type : str = type
+        
+    def to_DTO(self) -> CommandDefinitionDTO:
+        return CommandDefinitionDTO(
+            command_code=self.code,
+            description=self.description,
+            help_text=self.help_text,
+            response_type=str(self.response_type),
+            responses=self.responses,
+            #test_responses=self.test_responses,
```

Comment:
can see why you wouldnt want to return this, but it might be useful to see how many test responses are present?

Reply:
Maybe later when the API starts running tests, but the test responses are not currently needed by the API at the moment.

Reply:
Yeah it would more be useful for developers